pom.xml

```diff
@@ -1376,6 +1376,10 @@
 								<artifactId>frank-doc-frontend</artifactId>
 								<version>${frankdoc.version}</version>
 							</frontendArtifact>
+							<additionalJOptions>
```

Comment:
Ik denk dat je ook deze moet toevoegen: ` --add-opens java.base/java.io=ALL-UNNAMED`

Reply:
Bij mij lokaal en op de CI ging het goed :O 
Enig idee waar dit precies vandaan kwam? JDK 21?

Reply:
JDK17. Maar misschien dat het pas nodig is na mergen van changes die ik heb moeten doen in de branch van JJ, daar moest ik dit toevoegen aan de surefire JVM options, om een private field van een JDK class te kunnen zetten. (Ik kon geen andere oplossing vinden die volledig werkte).

Reply:
Ok, misschien dat dat niet meer hoeft? Ik liep ook tegen die --add-opens aan, zoals verwacht maar alleen deze 2 waren nodig.

Reply:
We zullen zien. Ik had dit PR in ieder geval ge-approved.

Reply:
Ik heb in Jenkins (build 7) alles op JDK17 gedraaid om te kijken of alles werkte en liep pas tegen problemen aan bij het draaien van Docker.

Reply:
Ja ik heb een probleem geintroduceerd in JJ's PR maar zag geen betere oplossing.
Ik liep een beetje op de zaken vooruit misschien.

Ik vroeg me al eerder af of we met meer "opens" meer issues in IAF-Test zouden kunnen fixen.

Voorlopig heeft de Maven build de extra "opens" niet nodig.

Reply:
Ik vind het wel goed dat we nog een comment toevoegen @nielsm5 waarom je deze 2 regels met extra `JOptions` hebt toegevoegd. Ik kan het nu ook niet raden eigenlijk en zou het je moeten vragen 😉